opencti-platform/opencti-graphql/tests/02-integration/05-parser/csv-parser-test.js

```diff
@@ -120,13 +120,13 @@ describe('CSV-PARSER', () => {
       .toBe(42);
     const relationshipTargets = objects.filter((o) => o.relationship_type === 'targets');
     expect(relationshipTargets.length)
-      .toBe(118);
+      .toBe(117);
     const relationshipLocatedAt = objects.filter((o) => o.relationship_type === 'located-at');
     expect(relationshipLocatedAt.length)
-      .toBe(130);
+      .toBe(125);
```

Comment:
why do we get less relationships in this test ?

Reply:
If i remember well its because the dedup is done on ids and as the id are generated now, its considered as duplicate.
Need to double check. @aHenryJard do you take a look on that in the PR review?

Reply:
Good point, I will look

Reply:
(I removed approval until I understand this point)

Reply:
Diff is legitimate because there was a before this PR
![image](https://github.com/user-attachments/assets/ccf54368-cec9-471a-b51d-7a5d48244c4a)